src/types.ts

```diff
@@ -241,6 +241,15 @@ export interface IUtxoFilterOptions {
   reward_lock?: number,
 }
 
+export type UtxoSelectionAlgorithm = (storage: IStorage, token: string, amount: number) => Promise<{ utxos: IUtxo[], amount: number }>;
+
+export interface IUtxoSelectionOptions {
+  token?: string,
+  changeAddress?: string,
+  chooseInputs?: boolean,
+  utxoSelectionMethod?: UtxoSelectionAlgorithm,
```

Comment:
I will create one because it will help with the organization later on, but we still have to decide how will this be configured by the wallet, or if the `SendTransaction` facade will receive the algorithm as an optional argument.